### DISCUSSION TITLE
posts/xpath/

### DISCUSSION THREAD

**Discussion by giscus** [marcosruiz/marcosruiz.github.io#11] [General]
# posts/xpath/

Introducción

https://marcosruiz.github.io/posts/xpath/

**Comment by rodrigomulero** [comment]
##El mejor testador de consultas XPath que he probado es 

https://extendsclass.com/xpath-tester.html

Se actualiza en tiempo real sobre la consulta que escribes sin necesidad de andar pulsando ningún botón.

##El más avanzado:

https://www.freeformatter.com/xpath-tester.html

Soporta consultas XPath 2.0 y 3.0
>A demás de devolver el tipo de salida de la consulta si es noto texto etc
>>"Include 'XML Item Type' in output"
